prow/tide/tide.go

```diff
@@ -1571,7 +1572,8 @@ func (c *syncController) presubmitsByPull(sp *subpool) (map[int][]config.Presubm
 			// - Brancher
 			// - RunBeforeMerge
 			// - Files changed
-			shouldRun, err := ps.ShouldRun(sp.branch, c.changedFiles.prChanges(&pr), ps.RunBeforeMerge, false)
+			force := (requireManuallyTriggeredJobs && ps.ContextRequired() && ps.NeedsExplicitTrigger()) || ps.RunBeforeMerge
```

Comment:
small NIT, I would rephrase the name to something like `forceRun` etc to be consistent with other names.